### ISSUE TITLE
Incorrect octave algorithm implementation

### ISSUE THREAD

**Issue by zaraken** [TheAlgorithms/MATLAB-Octave#31]
https://github.com/TheAlgorithms/MATLAB-Octave/blob/48e12faca2545389d04d1323cf7c7bbc7db28629/algorithms/Strings/isPalindrome.m#L17

This is obviously incorrect.
It's enough that the last pair of characters be the same to return true. However all pairs of characters need to be the same for the string to be considered a palindrome.

Would be nice if implementations came with tests.

**Comment by cozek** [comment]
You are right. Perhaps you could provide a correct implementation or give pointers on how to include some tests!
